cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -40,24 +31,23 @@
 
 
 @pytest.mark.parametrize(
-    "mapping_data_vm_obj_single_datastore",
+    "source_type, dest_type, template_type",
     [
-        ["nfs", "iscsi", rhel7_minimal],
-        ["iscsi", "iscsi", rhel7_minimal],
-        ["iscsi", "nfs", rhel7_minimal],
-        ["local", "nfs", rhel7_minimal],
-    ],
-    indirect=True,
+        ["nfs", "iscsi", Templates.RHEL7_MINIMAL.value],
```

Comment:
I'd like to avoid having to resolve the values on all of these in each parametrization, again its repetitive.

It can be resolved in the indirect fixture method that's using it.

Note if you use aenum NamedConstant/Constant class instead of base enum class, the value will be resolved automatically with `str(Template.RHEL7_MINIMAL)`

Reply:
Done